src/operations/booleans/intersect.test.js

```diff
@@ -170,6 +173,16 @@ test('intersect: intersect of one or more geom3 objects produces expected geomet
     [ [ 8, 8, 9 ], [ 8, 8, 8 ], [ 9, 8, 8 ], [ 9, 8, 9 ] ],
     [ [ 9, 9, 8 ], [ 9, 8, 8 ], [ 8, 8, 8 ], [ 8, 9, 8 ] ]
   ]
+*/
+  exp = [
```

Comment:
hmm how come there are more points than previously ?

Reply:
Six points in each case. Just the order changes due to the choice of bestplane in Node.js.